docs/dev/adding_instruments.rst

```diff
@@ -812,6 +819,122 @@ In the above example, :code:`MultimeterA` and :code:`MultimeterB` use a differen
 :code:`MultimeterB` can be defined subclassing :code:`MultimeterA` and just implementing the difference.
 
 
+.. _channels:
+
+Instruments with channels
+=========================
+
+Some instruments, like oscilloscopes and voltage sources, have channels whose commands differ only in the channel name. For this case, we have :class:`~pymeasure.instruments.Channel`, which is similar to :class:`~pymeasure.instruments.Instrument` and its property factories, but does expect an :code:`instrument` instead of an :code:`adapter` as parameter. All the channel communication is routed through the instrument's methods (`write`, `read`, etc.). However, :meth:`Channel.write <pymeasure.instruments.Channel.write>` uses `str.format` to replace any occurence of :code:`"{ch}"` with the channel's name in the command.
```

Comment:
It makes sense to have a new line for every sentence in  markdown/rst. This way, any diff is smaller/more localised, the lines are not endlessly long.

Reply:
> > (force push was just a rebase)
> 
> I'm not sure if rebasing breaks something in the Github flow, but I find it very hard to determine what has changed since the last time I viewed the PR, and thus assess the new changes. We could always rebase on merge, afaik that has the same effect. Or maybe not rebase on nearly every push? Or am I doing something wrong, here?

Yeah, I noticed, that it causes problems in the diff view. In this case, I wanted to include the documentation changes lest there are any conflicts.

Is merging the master branch into the feature branch better for the diff views?

I'll abstain from rebasing during a review, unless necessary.

Reply:
> Is merging the master branch into the feature branch better for the diff views?

I think so, at least I often see a "changes since you last viewed" thing, although I now can't swear if that can be used for reviewing or is just a diff view, again (then it wouldn't change much).

Reply:
I had a [comment](https://github.com/pymeasure/pymeasure/pull/718#discussion_r1017081240) queued in from the review, therefore the unresolving.

Reply:
Done